crates/templates/src/run.rs

```diff
@@ -405,6 +424,29 @@ impl Run {
             .build()
             .expect("can't fail due to no partials support")
     }
+
+    fn partials(&self) -> anyhow::Result<impl liquid::partials::PartialCompiler> {
+        let mut partials = Partials::empty();
+
+        if let Some(partials_dir) = self.template.partials_dir() {
+            for partial_file in std::fs::read_dir(partials_dir)? {
+                let partial_file = partial_file?;
+                if !partial_file.file_type().is_ok_and(|t| t.is_file()) {
+                    anyhow::bail!("Non-file in partials directory: {partial_file:?}");
```

Comment:
Why bail instead of just ignore?

Reply:
Because if the template author put non-files (e.g. directories) in the partials directory, then presumably they had some intent for those non-files, and we can't fulfil that intent.